### ISSUE TITLE
Some parts of the app are too dark when Fork is in the background

### ISSUE THREAD

**Issue by kitgrose** [ForkIssues/Tracker#56]
Compare the Fork with Finder, for instance:

![fork-background](https://user-images.githubusercontent.com/165977/27812583-46ac752c-60b3-11e7-93f1-a286731a3b99.png)

![finder-background](https://user-images.githubusercontent.com/165977/27812582-4652353a-60b3-11e7-8492-a22028423394.png)

Background tabs are especially noticeable (they seem to use the same colour, `rgb(179, 179, 179)`, whether the window is in the foreground or the background, which makes them far too dark when the window loses focus). Compare with Finder, where background tabs (`rgb(241, 241, 241)`) are very, very subtly distinct from the foreground tab (`rgb(246, 246, 246)`) when the window loses focus.

The highlighted repository is also shown in `rgb(170, 170, 170)` instead of `rgb(220, 220, 220)` like Finder uses.

**Comment by DanPristupov** [comment]
@kitgrose thanks for taking my attention to this. I updated the theme.
I did not use the Finder's color because I personally think that difference between active and other tabs are too shallow and I used slightly more contrast colors.

The changes will be available in the next update.

<img width="803" alt="2017-07-06 at 11 03" src="https://user-images.githubusercontent.com/618115/27903873-293a5624-623b-11e7-858f-653c9ece2543.png">
